Library/Homebrew/test/test_cmd_testbot.rb

```diff
@@ -0,0 +1,44 @@
+require "testing_env"
+require "dev-cmd/test-bot"
+
+class TestbotCommandTests < Homebrew::TestCase
+  def with_environment(h)
+    old = Hash[h.keys.map { |k| [k, ENV[k]] }]
+    ENV.update h
+    begin
+      yield
+    ensure
+      ENV.update old
+    end
+  end
+
+  def test_resolve_test_tap
+    tap = Homebrew::resolve_test_tap
```

Comment:
Prefer `Homebrew.resolve_test_tap` for invoking methods (should have been pointed out by `brew style`).